STYLEGUIDE.md

```diff
@@ -0,0 +1,101 @@
+- Internal links to sections should never rely on implicit references, since they may change with section titles or other re-organization. Instead, always use explicit references above the target with one blank line between; keep ref names short, use ``-`` to replace spaces, use unique and distinctive names and append ``-ref`` to the end of the name for clarity.
+- All external links should be inline (not bare) unless specifically needed to illustrate some property of the link itself, should be https if available, and should, generally, be written as a reference with `` `Test Link Name`_ `` and then included in full at after the end of the paragraph in which it appears, e.g. `` .. _Test Link Name: https://www.example.com/ `` . A block of links should be separated from paragraphs above and below by one blank line, with no blank lines between links. Make sure to check that your links, as the build will automatically check for broken ones. Strip anything unnecessary (parameters, referrers, etc) off the end. As the base docs are in US English, any links should be to the English, desktop version of the site.
+- Use enumerated lists with ``#. `` to list items with a clearly defined order, otherwise, use bulleted lists with ``* ``. Include a space between the bullet/number and the text. No blank line between elements, unless a sublist is present, but blank lines around the list. Sublist should have no spaces around elements. You should try to avoid more than two levels of lists if at all possible, and keep lists to between 3 and 10-20 elements; anything shorter should be written in prose, and anything longer should be simplified or broken into multiple lists/sublists.
+- Use ``note::`` for general callouts, ``warning::`` to warn users against procedures with potential serious consequences, and ``important::`` for key points that users don't want to miss. However, use these sparingly, only when clearly called for. Always make them separate blocks, with a blank line between the start of the block, like:
+```
+.. Note::
+
+   This is a note.
+```
+
+## Images/Figures
+
+- Organize images into directories following the Google drive
+- Lower case file names and extensions, separate with hyphens, 3-character extensions
+- For ease of visibility, explanatory images should be full width; expository or those with very simple subjects can be half-width, right aligned
```

Comment:
Why right aligned? Shouldn't they be left aligned?

Reply:
There may be another technical reason, but the layout flows better that way, and it makes conceptual sense for secondary, less important images to be such as they are supplementary and illustrative of the text, rather than the primary focus (or they'd be full width). That, and as I've emphasized multiple times its what is currently implemented in the docs; if you want to change it you can make an atomic PR after this one that changes it both here and in the docs themselves.